update.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+# Print what you're doing, exit on error.
+set -xe
+
+# Get the latest commit for the bundle. This is the string the bundle image is tagged with.
+get_tag () {
+    URL="https://api.github.com/repos/openshift/trustee-operator/commits?per_page=1&path=bundle"
+    curl "$URL" | jq -r '.[0].sha'
+}
+
+# Get the digest for a tagged image. Pass the <TAG> as the first argument.
+# Quay API docs are at https://docs.quay.io/api/swagger/#!/tag/listRepoTags.
+get_digest () {
+    TAG="$1"
+    URL="https://quay.io/api/v1/repository/redhat-user-workloads/ose-osc-tenant/trustee/trustee-operator-bundle/tag?specificTag=$TAG"
+    curl -L "$URL" | jq -r '.tags.[0].manifest_digest'
```

Comment:
shouldn't be '.tags[0].manifest_digest' instead? (without the '.' after tags)?
I got an error with current script:
```
++ jq -r '.tags.[0].manifest_digest'
jq: error: syntax error, unexpected '[', expecting FORMAT or QQSTRING_START (Unix shell quoting issues?) at <top-level>, line 1:
.tags.[0].manifest_digest      
jq: 1 compile error
```